hr_expense_advance_clearing/data/advance_product.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <data noupdate="1">
```

Comment:
'data noupdate="1"' can be removed as 'data' is deprecated and you can put 'noupdate="1"' within the odoo element like 'odoo noupdate="1"

Reply:
Done